### DISCUSSION TITLE
stream_select interrupted by an incoming signal

### DISCUSSION THREAD

**Discussion by dhodo15** [aldas/modbus-tcp-client#116] [Q&A]
I have try loops inside array of addresses that have length more than 300. 

Some adreses produce this error:
stream_select interrupted by an incoming signal

Is what I do wrong? Can u give an example by my case.

**Comment by aldas** [comment]
I assume this is something similar to https://github.com/php-amqplib/php-amqplib/issues/165   Do you happen to have long running loop? Does it happen always after similar length of time?

**Reply by dhodo15** [reply]
thx for reply. the scripts handling many device addresses and was run for every minute.  it always happen randomly, may it's happen cuz poor network? i have add more time out for request but its not effective..
